datacenter/dtr/2.2/guides/admin/configure/use-a-load-balancer.md

```diff
@@ -14,18 +15,45 @@ This allows users to access DTR using a centralized domain name. If a replica
 goes down, the load balancer can detect that and stop forwarding requests to
 it, so that the failure goes unnoticed by users.
 
-## Load-balancing DTR
+## Load balancing DTR
 
 DTR does not provide a load balancing service. You can use an on-premises
 or cloud-based load balancer to balance requests across multiple DTR replicas.
 
 Make sure you configure your load balancer to:
 
-* Load-balance TCP traffic on ports 80 and 443
+* Load balance TCP traffic on ports 80 and 443
 * Not terminate HTTPS connections
-* Use the `/health` endpoint on each DTR replica, to check if
-the replica is healthy and if it should remain on the load balancing pool or
-not
+* Use the `/health` endpoint (note the lack of an `/api/v0/` in the path) on each
+DTR replica, to check if the replica is healthy and if it should remain on the
+load balancing pool or not
+
+## Health check endpoints
+
+The `/health` endpoint returns a JSON object for the replica being queried with
```

Comment:
From what I can [see here](https://docs.docker.com/datacenter/dtr/2.2/reference/api/) the `/health` endpoint is not part of the swagger docs 😓
But for the `cluster_status` let's link, [here's the source](https://github.com/docker/docker.github.io/blob/master/datacenter/dtr/2.2/reference/api/index.html). I'm not sure we can do relative links in this case, since the swagger api is essentially an html app we just dump on that directory.